src/test/resources/data/typicalDataBundle.json

```diff
@@ -379,6 +384,30 @@
         "sessionLevel": {}
       }
     },
+    "instructor1OfCourseWithOnlyCustom": {
+      "googleId": null,
+      "courseId": "idOfCourseWithOnlyCustom",
+      "name": "Not registered instructor",
+      "email": "instructor@course.tmt",
+      "isArchived": false,
+      "role": "Custom",
+      "isDisplayedToStudents": true,
+      "displayedName": "Instructor",
+      "privileges": {
+        "courseLevel": {
+          "canviewstudentinsection": false,
+          "cansubmitsessioninsection": false,
+          "canmodifysessioncommentinsection": false,
+          "canmodifycourse": false,
+          "canviewsessioninsection": false,
+          "canmodifysession": false,
+          "canmodifystudent": false,
+          "canmodifyinstructor": true
+        },
+        "sectionLevel": {},
+        "sessionLevel": {}
+      }
+    },
```

Comment:
Do you need all these new test data?
If you understand what the method does, you should be able to make a test from existing data bundle.